jks/jks.py

```diff
@@ -125,10 +125,10 @@ def _read_data(data, pos):
 
 def _sun_jks_pkey_decrypt(data, password):
     'implements private key crypto algorithm used by JKS files'
-    password = ''.join([b'\0'+c.encode('latin-1') for c in password]) # the JKS algorithm uses a regular Java UTF16-BE string for the password, so insert 0 bytes
+    password = password.encode('UTF-16BE') # the JKS algorithm uses a regular Java UTF16-BE string for the password, so insert 0 bytes
     iv, data, check = data[:20], data[20:-20], data[-20:]
     xoring = zip(data, _jks_keystream(iv, password))
-    key = ''.join([chr(ord(a) ^ ord(b)) for a, b in xoring])
+    key = bytes([a ^ b for a, b in xoring])
```

Comment:
Well `a` and `b` are integers so it works
That's because iterating bytes in `_jks_keystream()` returns integers